<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="test1"
           tests="3"
           failures="1"
           errors="0"
           skipped="1"
           time="0.006">
  <properties />
            <testcase name="testMethodTwo" time="0.006" classname="com.odysii.test.impulse.helper.PassportSerialTest">
                <failure type="java.lang.AssertionError"
                  message="expected [true] but found [false]"
              >
<![CDATA[java.lang.AssertionError: expected [true] but found [false]
  org.testng.Assert.fail(Assert.java:93)
  org.testng.Assert.failNotEquals(Assert.java:512)
  org.testng.Assert.assertTrue(Assert.java:41)
  org.testng.Assert.assertTrue(Assert.java:51)
  addToBasket(SampleTest.java:14)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
  org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
]]>
      </failure>

          </testcase>
  
            <testcase name="testMethodThree" time="0.0" classname="com.odysii.test.impulse.helper.PassportSerialTest">
          <skipped />
    </testcase>
  
            <testcase name="testMethodOne" time="0.000" classname="com.odysii.test.impulse.helper.PassportSerialTest" />
      
</testsuite>
